client/js/store/modules/game.js

```diff
@@ -52,27 +53,56 @@ function handleGameResponse(context, jwres, resolve, reject) {
       return reject(jwres.body.message);
   }
 }
+
+/**
+ * @returns number of queens a given player has
+ * @param player is the player object
+ */
+function queenCount(player) {
+  return player?.faceCards?.reduce(
+    (queenCount, card) => queenCount + (card.rank === 12 ? 1 : 0),
+    0
+  );
+}
+
 const initialState = resetState();
 export default {
+  // TODO: Look in to namespacing -- currently this will not work due to a collision
+  // with the game socket also being named `game`. Potentially rename one or the other?
   state: initialState,
   getters: {
+    player(state) {
+      return state.players[state.myPNum];
+    },
+    playerPointTotal(state, getters) {
+      return getters.player?.points?.reduce((total, card) => total + card.rank, 0) || 0;
+    },
+    playerQueenCount(state, getters) {
+      return queenCount(getters.player);
+    },
```

Comment:
We lose a little bit of the organization here from the comments you had in GameView, @itsalaidbacklife, but I think the colocation more than makes up for it. Might be worth pulling some of those comments back here if you're really partial to them.

The big benefit here is that all components that need this data all get it from a single source of truth now and it means we don't need to do any sort of complex prop drilling.

Reply:
This is a solid improvement